lib/tasks/gettext.rake

```diff
@@ -52,6 +52,68 @@ namespace :gettext do
            files_to_translate.join(' '))
   end
 
+  def metadata_pot_file_path
```

Comment:
Can we move all this code into `lib/`? That way we can write some tests against the generated file.